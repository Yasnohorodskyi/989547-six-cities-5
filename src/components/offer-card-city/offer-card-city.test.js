import React from "react";
import renderer from "react-test-renderer";

import MainCard from "./offer-card-city";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

it(`Should main card render`, () => {
  const tree = renderer
  .create(<MainCard />).toJSON();

  expect(tree).toMatchSnapshot();
});
