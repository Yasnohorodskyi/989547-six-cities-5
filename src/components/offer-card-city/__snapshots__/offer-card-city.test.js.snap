// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should main card render 1`] = `
<OfferCard
  block="cities"
  className="cities__place-card"
  imgHeight="200"
  imgWidth="260"
/>
`;
